### ISSUE TITLE
Horizontal timelines and hours

### ISSUE THREAD

**Issue by Azmoinal** [seanlowe/obsidian-timelines#39]
It seems that horizontal timelines don't render correctly hours. 

I tried to use the inline template added using plugin command but it seems that any event, I introduced are rendered by midnight to midnight, ignoring the hour information. So basically the event below is collapsed in a line. 

```
<div class="ob-timelines"
	data-title="TestRange"
	data-description="Range"
	data-color=""
	data-type="range"
	data-start-date="1450-01-01-01"
	data-end-date="1450-01-01-16"
	data-tags="testtag">
	 Event description
</div>
```

I checked the documentation and it seems fine to me but I'm just starting to work on your plugin.

To render the horizontal timeline is used the following tags.

 ```
ob-timeline
 tags=korkinam
 startDate=1449
 endDate=1451
 minDate=1449
 maxDate=1451
 zoomInLimit=day
 zoomOutLimit=32140800000
 type=flat
 ```  

Is it a know limitation?

**Comment by seanlowe** [comment]
This has been fixed in release 2.1.6

Turns out I didn't correctly update the function that created the values vis-timeline needed for event dates to account for the hours when I redid the dates elsewhere.

Thanks for bringing this up

**Comment by Azmoinal** [comment]
This one just to confirm that the issue has been fixed. 
Thanks a lot for the very quick fix!
